### PR TITLE
Feature/cu mke411 fang ultimate abillity new

### PR DESCRIPTION
**Pull request by JoanMarcBardes**

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-mke411 Fang ultimate abillity](https://app.clickup.com/t/mke411)

File: Penteract/Assets/Scripts/Fang.cpp [hunk 1]
```diff
@@ -135,10 +149,26 @@ void Fang::CheckCoolDowns(bool noCooldownMode) {
...
+			ultimateCooldownRemaining = ultimateCooldown;
+			ultimateInCooldown = false;
+			Debug::Log("Ultiante ON");
+		}
+	}
 }
 
 void Fang::OnAnimationFinished() {
-	//TODO use if necesary
+	if (compAnimation->GetCurrentState()) {
```
Comment (Jordi-Gil) [inline Penteract/Assets/Scripts/Fang.cpp:164]: Check compAnimation nullity. Here and check the rest of functions

**Review by Jordi-Gil — APPROVED** [review]

**Review by jcarlos0305 — DISMISSED** [review]

File: Penteract/Assets/Scripts/UltimateFang.cpp [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			Debug::Log(collidedWith.name.c_str());
+		}
+	}
+}
+
+void UltimateFang::StartUltiamte()
+{
+	ComponentSphereCollider* sphereCollider = GetOwner().GetComponent<ComponentSphereCollider>();
+	sphereCollider->radius = radius;
+	//App->physics->UpdateSphereRigidbody(sphereCollider); //TODO We need this function to be exposed, then we can use it
```
Comment (Mefiso) [inline Penteract/Assets/Scripts/UltimateFang.cpp:77]: I think it is already exposed in the current version of develop?

Comment (Jordi-Gil) [reply Penteract/Assets/Scripts/UltimateFang.cpp:77]: Noice

**Review by Jordi-Gil — APPROVED** [review]

**Review by jcarlos0305 — APPROVED** [review]

**Review by devildrake — APPROVED** [review]

**Pull request merged**